SolastaUIUpdates/SolastaUIUpdates/Viewers/ModSettingsViewer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityModManagerNet;
+using ModKit;
+
+namespace SolastaUIUpdates.Viewers
+{
+    public class ModSettingsViewer : IMenuSelectablePage
+    {
+        public string Name => "Settings";
+
+        public int Priority => 1;
+
+        private static void DisplaySpellPanelSettings()
+        {
+            UI.Label("");
+            UI.Label("Spells Panel".yellow().bold());
+
+            int intValue = Main.Settings.MaxSpellLevelsPerLine;
+            if (UI.Slider("Max Levels per line", ref intValue, 3, 7, 5, "", UI.AutoWidth()))
+            {
+                Main.Settings.MaxSpellLevelsPerLine = intValue;
+            }
+
+            float floatValue = Main.Settings.SpellPanelGapBetweenLines;
+            if (UI.Slider("Gap between spell lines", ref floatValue, 30f, 50f, 30f, 0, "", UI.AutoWidth()))
```

Comment:
I originally had the vertical spacing smaller, set to 10. I wonder if that should be the minimum.